python/ray/util/sgd/torch/torch_trainer.py

```diff
@@ -119,8 +113,12 @@ def setup(self, config):
             ``step`` will be called after every optimizer step. If "epoch",
             ``step`` will be called after one pass of the DataLoader. If
             "manual", the scheduler will not be incremented automatically -
-            you are expected to call ``trainer.update_schedulers`` manually.
+            you are expected to call ``trainer.update_scheduler`` manually.
             If a scheduler is passed in, this value is expected to not be None.
+        use_local (bool): If True, 1 worker will be a local worker running
+            on the driver process, and all other workers will be remote. If
+            False, all workers will be remote. Set this to True for easy
+            debugging of worker on driver process. Defaults to False.
```

Comment:
Set this to True for easy debugging of worker on driver process.

mention that this mode can actually cause issues with cuda devices.